### PR TITLE
Configured Jest and Enzyme

### PR DESCRIPTION
**Pull request by P-1-X-3-L**

### CODE REVIEW

**Review by kishore0079 — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -34,5 +38,11 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.5",
```
Comment (pranavakkian) [inline package.json:44]: This is not going to work with react 17. Find out the appropriate version.

File: package.json [hunk 2]
```diff
@@ -1,4 +1,8 @@
-{
+{"jest": {
+  "snapshotSerializers": ["enzyme-to-json/serializer"],
+  "collectCoverageFrom": ["src/**/*.js", "!src/index.js"],
+  "coverageReporters": ["text"]
```
Comment (pranavakkian) [inline package.json]: This is not required. Instead add code coverage constraint. Find out on google on how to add global code coverage threshold on jest.
```suggestion
```

File: package.json [hunk 3]
```diff
@@ -1,4 +1,8 @@
-{
+{"jest": {
```
Comment (pranavakkian) [inline package.json]: keep this part after dev-dependencies. Also maintain proper indentation.

**Review by pranavakkian — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (shivalibaghel7) [inline package.json]: Formatting is not proper.

File: package.json (hunk 2 above)
Comment (shivalibaghel7) [inline package.json]: For threshold, ensure 100% test coverage.

**Review by shivalibaghel7 — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -34,5 +35,23 @@
...
+  },
+  "jest": {
+    "snapshotSerializers": [
+      "enzyme-to-json/serializer"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 100,
+        "functions": 100,
+        "lines": 100
```
Comment (pranavakkian) [inline package.json]:
```suggestion
        "branches": 100,
        "functions": 100,
        "lines": 100,
        "statements": 100 
```

**Review by pranavakkian — CHANGES_REQUESTED** [review]

File: src/setupTests.js [hunk 5]
```diff
@@ -3,3 +3,6 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
```
Comment (shivalibaghel7) [inline src/setupTests.js]: Remove comments.

**Review by shivalibaghel7 — CHANGES_REQUESTED** [review]

**Review by pranavakkian — DISMISSED** [review]

**Review by pranavakkian — APPROVED** [review]

**Review by shivalibaghel7 — APPROVED** [review]

**Review by kishore0079 — APPROVED** [review]

**Pull request merged**